src/components/profile/edit/EditPersonalInfo.vue

```diff
@@ -34,6 +34,7 @@
           :avatar="avatar"
           :size="264"
           :isStaff="scope.isStaff"
+          :edit="true"
```

Comment:
Why aren't we just removing this icon entirely from the editpersonalinfo.vue file? If we are hiding the picture always, why do we need to add a boolean flag to the userpicture?
Also, it doesn't seem appropriate that the UserPicture knows anything about any editing mechanism of the profile at the moment.

Reply:
We could also rename the `isStaff` attribute into `showLabel` and just use that for everything. Thoughts?